udata_front/theme/gouvfr/datagouv-components/src/components/DataserviceCard/DataserviceCard.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <article class="fr-my-2w fr-p-2w border border-default-grey fr-enlarge-link" :style="props.style">
+    <div class="absolute top-0 fr-grid-row fr-grid-row--middle fr-mt-n3v fr-ml-n1v" v-if="dataservice.private">
+      <p class="fr-badge fr-badge--sm fr-badge--mention-grey text-grey-380 fr-mr-1w">
+        <span class="fr-icon-lock-line fr-icon--sm" aria-hidden="true"></span>
+        {{ t('Private') }}
+      </p>
+    </div>
+    <div class="fr-grid-row fr-grid-row--gutters fr-grid-row--top">
+      <div class="fr-col">
+        <h4 class="fr-text--md fr-mb-1v fr-grid-row">
+          <slot name="dataserviceUrl" :dataservice="dataservice" :dataserviceUrl="dataserviceUrl">
+            <span class="fr-icon-terminal-line fr-icon--sm fr-mr-1v" aria-hidden="true"></span>
+            <AppLink :to="dataserviceUrl" class="text-grey-500">
+              {{ dataservice.title }}
+            </AppLink>
+          </slot>
+        </h4>
+        <div class="fr-text--sm fr-mb-0" v-if="dataservice.organization || dataservice.owner">
+          <template v-if="dataservice.organization">
+            <span class="not-enlarged dash-after-sm fr-mr-1v">
+              <AppLink class="fr-link fr-text--sm" v-if="organizationUrl" :to="organizationUrl">
+                <OrganizationNameWithCertificate :organization="dataservice.organization" />
+              </AppLink>
+              <OrganizationNameWithCertificate v-else :organization="dataservice.organization" />
+            </span>
+          </template>
+          <span class="not-enlarged dash-after-sm fr-mr-1v" v-else>
+            {{ ownerName }}
+          </span>
+          <span class="text-mention-grey">{{ t('Updated {date}', {date: formatRelativeIfRecentDate(dataservice.metadata_modified_at)}) }}</span>
+        </div>
+        <div class="fr-grid-row fr-grid-row--middle fr-text--sm text-mention-grey">
+          <p class="fr-my-0"><span class="fr-icon-information-line fr-icon--sm text-grey-380 fr-mr-1v"></span></p>
+          <p class="fr-text--sm fr-mr-1v fr-my-0">{{ t('Availability :') }}</p>
+          <p class="text-grey-400 fr-text--sm fr-my-0">{{ t('{n}% on the last month', dataservice.availability) }}</p>
```

Comment:
```suggestion
          <p class="text-grey-400 fr-text--sm fr-my-0">{{ t('{n}%', dataservice.availability) }}</p>
```

We only want to show the number (even if its shown on Figma)